analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```diff
@@ -144,10 +147,57 @@ public void Verify_ParseOptions()
                 .WithMessage("CSharp5: Unexpected build error [CS8026]: Feature 'target-typed object creation' is not available in C# 5. Please use language version 9.0 or greater. on line 3");
         }
 
+        [TestMethod]
+        public void Verify_ErrorBehavior()
+        {
+            var builder = WithSnippet("undefined");
+            builder.Invoking(x => x.Verify()).Should().Throw<UnexpectedDiagnosticException>()
+                .WithMessage("CSharp7: Unexpected build error [CS0116]: A namespace cannot directly contain members such as fields, methods or statements on line 1");
```

Comment:
pedantic: alignment if off

Reply:
It is meant to be like this according to the `How we work` from 2021-12-22. It's not documented yet, I have an action item to document it.

Reply:
My understanding was that we discussed "method declaration arguments" and not about this. I find aligned indentations a lot easier to read.

Reply:
We've discussed both, see notes.
_Method declaration:_ We've decided not to change anything. Mary will investigate VS tools.
_Identation on chained invocation calls:_ This is the one that applies here and that I have the action item for to document it.